modules/services/window-managers/i3-sway/sway.nix

```diff
@@ -436,7 +437,7 @@ in {
         onChange = ''
           swaySocket="''${XDG_RUNTIME_DIR:-/run/user/$UID}/sway-ipc.$UID.$(${pkgs.procps}/bin/pgrep --uid $UID -x sway || true).sock"
           if [ -S "$swaySocket" ]; then
-            ${pkgs.sway}/bin/swaymsg -s $swaySocket reload
+            ${swayPackage}/bin/swaymsg -s $swaySocket reload
```

Comment:
Can the `if else` be inlined here or the variable scoped to the `onChange` script?
Currently the variable is used once and declared 400 lines before its use.

Reply:
Thanks for the review/suggestion! I tried inlining it, but ./format wanted the `${}` to be spaced out over a few lines, so I applied your second idea of scoping the var instead.